src/Skoruba.IdentityServer4.Admin/Configuration/Users.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Skoruba.IdentityServer4.Admin.Configuration
+{
+    public class Users
+    {
+        public const string AdminUserName = "bob";
+        public const string AdminPassword = "Pa$$word123";
+        public const string AdminEmail = "BobSmith@email.com";
```

Comment:
Make sure this is not a real email or upon connection with an email server, someone will get spammed. :-) I suggest using @example.com or creating a site for the repo and a demo account inbox on it.